astropy/table/tests/test_index.py

```diff
@@ -443,6 +443,17 @@ def test_table_loc_indices(self, main_col, table_types, engine):
         for i, p in zip(t2, [1, 4, 2]):  # same order as input list
             assert i == p - 1
 
+    def test_multiple_index_loc(self, main_col, table_types):
+        # see https://github.com/astropy/astropy/issues/13176
+        if isinstance(main_col, Time):
+            pytest.skip()
```

Comment:
I don't know enough about whether this case should be supported too, and since it doesn't pass the test yet, I just skip it. I can look more into it if needed.